### PR TITLE
转换规则 No. 219/230

### PR DESCRIPTION
**Pull request by co63oc**
### PR Docs

https://github.com/PaddlePaddle/PaConvert/issues/112
219 torch.profiler.schedule 已有文档验证无误
230 torch.sparse.softmax 已有文档验证无误

### PR APIs

```bash

```

### CODE REVIEW

File: paconvert/api_matcher.py [hunk 1]
```diff
@@ -694,6 +694,18 @@ def generate_code(self, kwargs):
         return GenericMatcher.generate_code(self, kwargs)
 
 
+class SparseSoftmaxMatcher(BaseMatcher):
+    def generate_code(self, kwargs):
+        code = ""
+        if "dtype" in kwargs:
+            dtype_v = kwargs.pop("dtype")
+            code = code + "{}=paddle.sparse.cast({}, value_dtype={})\n".format(
```
Comment (zhwesky2010) [inline paconvert/api_matcher.py:702]: 建议换一种写法，用一个临时值来表达，sparse.cast的输出。因为输入可能是一个运算表达式，例如`torch.sparse.softmax(x*2)`

File: tests/test_sparse_softmax.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_5():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        indices = [[0, 1, 2], [1, 2, 0]]
+        values = [1., 2., 3.]
+        x = torch.sparse_coo_tensor(indices, values, [3, 3])
```
[the remark this answers is not in the document]
Comment (co63oc) [reply tests/test_sparse_softmax.py:84]: 图片
torch不支持CSR格式

File: paconvert/api_matcher.py (hunk 1 above)
Comment (co63oc) [reply paconvert/api_matcher.py:703]: 已修改

**Review by zhwesky2010 — APPROVED** [review]
LGTM
